specfile/tags.py

```diff
@@ -499,6 +499,10 @@ def regex_pattern(tag):
         data = []
         buffer: List[str] = []
         for line, valid in lines:
+            ws = ""
+            tokens = re.split(r"([^\S\n]+)$", line, maxsplit=1)
+            if len(tokens) > 1:
+                line, ws, _ = tokens
```

Comment:
No, it can't. [`re.split()`](https://docs.python.org/3/library/re.html#re.split) with `maxsplit=1` and a group in the pattern always returns 1 (if there is no match) or 3 tokens (the last being an empty remainder).

Reply:
With Python 3.7+ it can even be written in a way that it always returns 3 tokens, so there is no need for the condition and it can be a one-liner, but *specfile* still supports Python 3.6 :slightly_smiling_face: